src/timepicker/styles/timePicker.m.css

```diff
@@ -0,0 +1,77 @@
+@import '../../common/styles/variables.css';
+
+.root,
+.root * {
+	box-sizing: border-box;
+}
+
+.root {
+	display: inline-block;
+	position: relative;
+	width: var(--form-field-width);
```

Comment:
Do you think that fields should have a width, or that they should expand to take up the space they are given?